tasks/spam.js

```diff
@@ -7,7 +7,7 @@ const usernoteHelper = require('../services/usernote-helper');
 const SUBREDDIT = 'pokemontrades';
 
 const SPAM_COOLDOWN = 60 * 60 * 6; // The number of seconds allowed between threads
-const TRADE_TYPE_CSS_CLASSES = new Set(['event', 'shiny', 'comp', 'casual', 'item', 'tradeback', 'dexevo', 'file', 'redeem', 'bank', 'xyoras', 'letsgo']);
+const TRADE_TYPE_CSS_CLASSES = new Set(['tradeback', 'dexevo', 'redeem', 'swsh', 'smusum', 'xyoras', 'letsgo']);
```

Comment:
Should we remove dexevo and contest while we're at it?

What if we used a more futureproof solution so that we don't have to edit this every time we edit flairs? I think we could use double css classes in the link flairs for this. For example:

```suggestion
const TRADE_TYPE_CSS_CLASSES = new Set(['trade']);
```

Then we just add `trade` to the end of all the link flair css classes on reddit.

What do you think?

Reply:
> Should we remove `dexevo` and `contest` while we're at it? And add `home`?

`dexevo` is the Tradeback flair's CSS class. We kept it for compatibility with older Tradeback topics when we switched the tag. Good call on the other two though!

> Actually, what if we used a more futureproof solution so that we don't have to edit this every time we edit flairs?

I'm very mixed on this. I would consider it a great idea, if not for the fact that I'd rather avoid adding more dependence on CSS classes.

This whole thing actually reminded me to go back and see what happened with our old topics after our recent CSS changes. Sure enough, it's rather inconsistent: ![](https://i.imgur.com/Ah6w4SU.png)

Interestingly enough, if we had not had CSS classes, those old tags would've retained their color, just as they still do on new Reddit: ![](https://user-images.githubusercontent.com/9310656/68999814-1775ce80-0894-11ea-83da-a65bc2f24699.png)

The reason for this is that if a CSS class is present, Reddit will drop the rich text formatting for that flair while it is being viewed from the subreddit. If we drop the CSS class, though, the color will be stored with future topics, and we can still apply the rest of our styling options since they are consistent for all the flairs.

I've added a commit that switches from relying on the CSS classes to the flair text itself so that we have the option of removing the CSS classes.